Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.12.8-alpine
+LABEL maintainer="sberdianskyi@mate.com"
+ENV PYTHONBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+
+RUN mkdir -p /files/media
```

Comment:
Consider combining the `RUN mkdir -p /files/media` and `RUN chown -R django_user /files/media` into a single `RUN` command to reduce the number of layers in the Docker image.